### DISCUSSION TITLE
what kind of updates you use this for?

### DISCUSSION THREAD

**Discussion by zivian** [fwupd/fwupd#3398] [Q&A]
i am guessing this is used for bios updates, proprietaty drivers/blobs.

am i correct? if no, where can i find more information?

**Comment by hughsie** [comment]
We do probably need a better introduction in the README.md file. What about something like:

    The fwupd project is designed to deploy persistent firmware onto devices. This doesn't include firmware loaded every time the device starts up (see linux-firmware for that).

I'm rubbish at prose like this.
